src/View/Helper/FlashMessenger.php

```diff
@@ -245,9 +253,14 @@ public function getAutoEscape()
      * @param  string $messageCloseString
      * @return FlashMessenger
      */
-    public function setMessageCloseString($messageCloseString)
+    public function setMessageCloseString($messageCloseString, string $namespace = 'default')
```

Comment:
We should probably have  the string `'default'` in a `private const`, to prevent copy-paste mistakes, and ease future refactoring

Reply:
As I've understood `'default'` is are namespace name from `Laminas\Mvc\Plugin\FlashMessenger\FlashMessenge::NAMESPACE_DEFAULT`